packages/shared/components/tiles/tileContents/AccountActivityTileContent.svelte

```diff
@@ -1,17 +1,17 @@
 <script lang="ts">
     import { localize } from '@core/i18n'
-    import { AliasActivity, getActivityTileTitle } from '@core/wallet'
+    import { AccountActivity, getActivityTileTitle } from '@core/wallet'
     import { truncateString } from '@core/utils'
     import { ActivityTileContent } from 'shared/components'
     import { Icon } from '@lib/auxiliary/icon'
 
-    export let activity: AliasActivity
+    export let activity: AccountActivity
 
     $: action = localize(getActivityTileTitle(activity))
     $: formattedAsset = {
-        text: truncateString(activity.aliasId, 6, 6),
+        text: truncateString(activity.accountId, 6, 6),
```

Comment:
same. I think that for the user it will be clearer to see the address of the account than the id